packages/caravan-wallets/src/trezor.ts

```diff
@@ -118,9 +117,7 @@ const ENV_TREZOR_BLOCKBOOK_URL =
 
 const TREZOR_CONNECT_URL =
   ENV_TREZOR_CONNECT_URL ||
-  (typeof window === "undefined"
-    ? ""
-    : `https://${window?.location.hostname}:8088/`);
+  (typeof window === "undefined" ? "" : "https://connect.trezor.io/9.6.0/");
```

Comment:
this change got local development working with a local blockbook instance (along with setting TREZOR_DEV=true)